app/controllers/application_controller.rb

```diff
@@ -3,4 +3,5 @@
 # Controller Application
 class ApplicationController < ActionController::Base
   include SessionsHelper
+  include UsersHelper
```

Comment:
Include should be used where it is needed.
Should not be used at base controller.
This will cause all controllers to load UsersHelper.